x-pack/legacy/plugins/ml/public/components/create_job_link_card/create_job_link_card.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import {
+  // @ts-ignore
+  EuiCard,
+  EuiIcon,
+  IconType,
+} from '@elastic/eui';
+
+interface Props {
+  iconType: IconType;
+  title: string;
+  description: string;
+  onClick(): void;
+}
+
+// Component for rendering a card which links to the Create Job page, displaying an
+// icon, card title, description and link.
+export const CreateJobLinkCard: FC<Props> = ({ iconType, title, description, onClick }) => {
```

Comment:
This could just be an implicit return. 
```
() => (<EuiCard...);
```

Reply:
Done in https://github.com/elastic/kibana/pull/42685/commits/31da8259e659eb908cba4922d4ab44bd0dd3898c